### PR TITLE
Added Node 9 and removed Node 7 from CI

### PR DESCRIPTION
**Pull request by delvedor**
As titled.

### CODE REVIEW

**Comment by StarpTech** [comment]
Looks like that Node 9 is too new for their build system.

**Comment by StarpTech** [comment]
This can take some days.

**Comment by StarpTech** [comment]
@delvedor https://github.com/appveyor/ci/issues/1895 Just deployed :D try again!

**Comment by delvedor** [comment]
I don't have an account for AppVeyor, can you rerun the build?

**Comment by StarpTech** [comment]
I don't have access to this repo.

File: appveyor.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ environment:
     - nodejs_version: 6
       platform: x64
       msvs_toolset: 12
-    - nodejs_version: 7
+    - nodejs_version: 8
```
Comment (StarpTech) [inline appveyor.yml]: Add

```
skip_commits:
  files:
    - docs/*
    - travis.yml
    - '**/*.html'
    - '*.md'
```
very useful only supported on appveyor

**Comment by mcollina** [comment]
I've rerun the build.

File: appveyor.yml (hunk 1 above)
Comment (mcollina) [reply appveyor.yml]: can you send a PR for this?

**Comment by mcollina** [comment]
CI is green, landing.

File: appveyor.yml (hunk 1 above)
Comment (StarpTech) [reply appveyor.yml]: yes, I can.

**Pull request merged**